docs/install.rst

```diff
@@ -304,6 +304,18 @@ If you have already activated the correct environment, but you still do not see
 Frequently asked questions
 --------------------------
 
+**When importing lumicks.pylake on Windows I get an error with cv2**
+
+The full error message is::
+
+    ImportError: DLL load failed while importing cv2: The specified procedure could not be found.
+    
+To resolve this error, follow the following steps:
+    
+* Remove the current Pylake installation. If you are not using Anaconda for anything else but Pylake, you can simply remove Anaconda in this step.
```

Comment:
Are we sure removing only Pylake is sufficient?

Reply:
No.... I will just say to remove Anaconda then.

Reply:
👍